sdk/template/azure-template/setup.py

```diff
@@ -18,32 +20,46 @@
 # a-b-c => a.b.c
 namespace_name = PACKAGE_NAME.replace('-', '.')
 
+# azure v0.x is not compatible with this package
+# azure v0.x used to have a __version__ attribute (newer versions don't)
+try:
+    import azure
+    try:
+        ver = azure.__version__
+        raise Exception(
+            'This package is incompatible with azure=={}. '.format(ver) +
+            'Uninstall it with "pip uninstall azure".'
+        )
+    except AttributeError:
+        pass
+except ImportError:
+    pass
+
 # Version extraction inspired from 'requests'
 with open(os.path.join(package_folder_path, 'version.py'), 'r') as fd:
     version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
                         fd.read(), re.MULTILINE).group(1)
+
 if not version:
     raise RuntimeError('Cannot find version information')
 
-with open('README.md', encoding='utf-8') as f:
-    long_description = f.read()
+with open('README.rst', encoding='utf-8') as f:
```

Comment:
Didn't we want to stick with Markdown for everything? I seem to recall that.

Reply:
Your template should have a annotation to not be updated automatically by the bot ;)

Reply:
`autoupdate=false` in setup.cfg right? Damnit did I not check in that change? *screaming